### ISSUE TITLE
Is there a limit to the number of routes

### ISSUE THREAD

**Issue by brightpixels** [sandulat/routes-gen#23]
### Describe the bug

In a Remix app, we are struggling to add any additional routes. We are getting the following error:
```
SyntaxError: Unexpected end of JSON input
    at JSON.parse (<anonymous>)
    at /Users/my-appl/node_modules/@routes-gen/remix/dist/index.js:52:30
    at ChildProcess.exithandler (child_process.js:310:7)
    at ChildProcess.emit (events.js:376:20)
    at maybeClose (internal/child_process.js:1055:16)
    at Socket.<anonymous> (internal/child_process.js:441:11)
    at Socket.emit (events.js:376:20)
    at Pipe.<anonymous> (net.js:673:12)
```

The only way around it, is to delete an existing route in order to create a new one.

I have tested on a fresh remix app and limit appears to be 68 routes and issues shows up when I create any additional routes.

### Your Example Website or App

Local app

### Steps to Reproduce the Bug or Issue

1. Create 70 routes on a fresh remix app 
2. Try to run the command `routes-gen -d @routes-gen/remix` in a Remix App 

### Expected behavior

To be able to generate an updated /routes.d.ts file

### Platform

- OS: macOS

**Comment by sandulat** [comment]
@brightpixels Looking into it.

**Comment by sandulat** [comment]
@brightpixels This is now fixed and published under `@routes-gen/remix` `0.3.3`.
<img width="549" alt="image" src="https://user-images.githubusercontent.com/7345874/183248710-6e4b98ae-a3ea-49f4-9f9a-745ef8028eb8.png">

Upgrade with `yarn add @routes-gen/remix@latest`. 

Please let me know if you still encounter this issue.

Thank you very much for opening the issue!
